rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -66,6 +70,7 @@ class PackageNameConverter {
   // For example: nflx-djangobase-enhanced-0.1-h12.170cdbd.all
   public static OsPackageName parseRpmPackageName(String fullyQualifiedPackageName) {
     OsPackageName osPackageName = new OsPackageName()
+    String versionDelimiter = BakeRequest.PackageType.RPM.getVersionDelimiter()
```

Comment:
Noticed that you refer to `versionDelimiter` up above against `DEB`, but used `getVersionDelimiter()` here for `RPM`.